components/compliance-service/ingest/server/compliance.go

```diff
@@ -38,7 +39,7 @@ type ComplianceIngestServer struct {
 	mgrClient          manager.NodeManagerServiceClient
 	automateURL        string
 	notifierClient     notifier.Notifier
-	updateManager      *projectupdater.Manager
+	updateManager      *project_update_lib.DomainProjectUpdateManager
```

Comment:
Removed the specific compliance project update manager to use the common domain project update manager